dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -33,6 +33,9 @@ ssoDomainNames.desc=A pipe (|) separated list of domain names that SSO Cookies s
 ssoUseDomainFromURL=Use domain name from the request URL
 ssoUseDomainFromURL.desc=Specifies whether to use the domain name from the request URL for the cookie domain.
 
+useSSOForJaspic=Enable single sign-on for JASPIC
+useSSOForJaspic.desc=Specifies whether single sign-on is enabled for a JASPIC authentication.
+
 ssoRequiresSSL=Single sign-on requires SSL
 ssoRequiresSSL.desc=Specifies whether a SSO cookie is sent over SSL. The equivalent property in the full application server profile is requiresSSL.
```

Comment:
Suggestion:
Specifies whether an SSO cookie is sent over SSL. The equivalent property in the full application server profile is requiresSSL.

Reply:
@bsbyrd1 Hi. Thank you for the review. There might have been a misunderstanding. The new property to review is useSSOForJaspic and the message reviewed was ssoRequiresSSL.